### ISSUE TITLE
spring-cloud-starter-netflix-eureka-client intermittently crashes

### ISSUE THREAD

**Issue by SledgeHammer01** [spring-cloud/spring-cloud-netflix#3759]
[hs_err_pid11908.log](https://github.com/spring-cloud/spring-cloud-netflix/files/4367393/hs_err_pid11908.log)

Windows 10 Pro x64
JDK 11.0.6 x64
Spring Tool Suite 4 (did a check for updates, I'm up to date, no errors)
Spring Boot 2.2.5
I have a Workspace, it contains 3 projects.

my app (Spring Boot 2.2.5)
Eureka server (Spring Boot 2.2.5)
my test app (Spring Boot 2.2.5)
I start the Eureka server. Then I hit the run button to "my app" and intermittently, it will immediately terminate. The console says nothing but "<terminated> - my app.....".

This is what I have done so far:

completely uninstall all JDKs and install JDK 11.0.6
completely remove STS4 and all remnants of anything Java related, the .m2 directory, etc.
completely removed the workspace and re-created it from scratch from github
I've ran Clean Projects countless times
I've ran mvnw clean install and mvnw clean package countless times
I've got no error messages or nothing, all STS4 tells me is that it terminated.

Seems to be related to spring-cloud-starter-netflix-eureka-client 2.2.2.RELEASE dependency. If I remove that from my POM and comment out the @EnableEurekaClient, then it starts 100% of the time.

One in a million tries, I get the error log (attached). Seems to be a class not found exception.

If I remove the dependency, my app starts 100% of the time no problem.

**Comment by SledgeHammer01** [comment]
Some more oddness:

1) as is right  now, it starts maybe 1 in 5 times
2) from command line it starts 100% of the time
3) mvnw clean install and package starts 100% of the time
4) I'm working from home, and I noticed when I am connected to the VPN it starts 100% of the time, even in STS
5) the crash dump gave me a hint since its crashing @ findFirstNonLoopbackHostInfo()... right now my client points to eureka01.xxxsoftware.com:8761 which is in the HOSTS file pointing to 127.0.0.1 which is a loopback. I tried pointing to 192.168.1.100 which is my PC IP and it seems to start 100% of the time. Not sure why 127.0.0.1 would be intermittent.
